src/comments.js

```diff
@@ -933,6 +938,31 @@ function printComment(commentPath, options) {
   }
 }
 
+function isJsDocComment(comment) {
+  const lines = comment.value.split("\n");
+  return (
+    lines.length > 1 &&
+    lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*")
+  );
+}
+
+function printJsDocComment(comment) {
+  const lines = comment.value.split("\n");
+
+  return docBuilders.concat([
+    "/*",
+    join(
+      docBuilders.hardline,
+      lines.map(
+        (line, index) =>
+          (index > 0 ? " " : "") +
+          (index < lines.length - 1 ? line.trim() : line.trimLeft())
```

Comment:
Why can’t we `.trim()` the last line?

Reply:
@lydell See my comment [here](https://github.com/prettier/prettier/pull/2470#discussion_r143339577). If we're okay with that behavior, then we can `.trim()` the last line.

Reply:
Sorry, I missed that comment. Then you are correct – we should not trim the right side on the last line!